_applications/project-jiminy.adoc

```diff
@@ -0,0 +1,344 @@
+with the following command:
+
+....
+oc create -f https://raw.githubusercontent.com/radanalyticsio/jiminy-tools/master/openshift-templates/data-store-loader.yaml
+....
+
+This will create the necessary Job object in OpenShift. To start the loader
+process run this command:
+
+....
+oc new-app \
+   -p JOB_NAME=jiminy-data-loader \
+   -p DB_HOST=postgresql \
+   -p DB_USER=postgres \
+   -p DB_PASSWORD=postgres \
+   -p DB_DBNAME=postgres \
+   -p DATASET_URL=http://files.grouplens.org/datasets/movielens/ml-latest-small.zip \
+   jiminy-data-loader
+....
+
+A Job has now been started that will load your data store with the MovieLens
+data. It will download the data files from the URL given in the `DATASET_URL`
+parameter. This command is loading the small data which only contains
+100,000 ratings on 9,000 movies from 700 users. This set is a nice baseline
+to start with as it will only take a minute or two to load. If you would like to
+test a larger data set, simply change the `DATASET_URL` parameter to
+`pass:[http://files.grouplens.org/datasets/movielens/ml-latest.zip]`. Please
+note that the larger data set will take several minutes to complete.
+
+You might want to check the progress of the data loader to ensure that it has
+completed. To do this, run the following command to inspect the logs:
+
+....
+oc logs -f job/jiminy-data-loader
+....
+
+If it has succeeded you should see an output like this:
+
+....
+INFO:root:starting data loader
+INFO:root:connecting to database
+INFO:root:creating products table
+INFO:root:creating ratings table
+INFO:root:downloading and unzipping dataset
+INFO:root:found movies.csv file in dataset
+INFO:root:found ratings.csv file in dataset
+INFO:root:loading products table
+INFO:root:loading ratings table
+....
+
+==== Launch MongoDB Model Store
+
+This database will host the trained models that the predictor service will
+utilize to make its recommendations. To start it from the command line, run
+the following command:
+
+....
+oc new-app \
+   -e MONGODB_USER=mongo \
+   -e MONGODB_PASSWORD=mongo \
+   -e MONGODB_DATABASE=models \
+   -e MONGODB_ADMIN_PASSWORD=mongoadmin \
+   --name mongodb \
+   centos/mongodb-26-centos7
+....
+
+This command will create the database using the parameters specified in the
+environment variables. The `MONGODB_DATABASE` variable must be set to `models`
+for this version of the Jiminy suite to ensure that the modeler and predictor
+have access to the proper storage.
+
+****
+The values shown here are simple defaults. If you change these values be
+sure to record the new values as you will need them to spawn the connected
+services.
+****
+
+=== Create application services
+
+With the storage services running, you can now deploy the application services
+which will perform the analytics and serve the user interface. _Now is a
+good time to ensure that you have performed the
+link:#prerequisites[prerequisite] oshinko install._
```

Comment:
Is this link working? It sends me to a blank page. ?missing an 's' [prerequisites]